### ISSUE TITLE
ONNX Docker: в таблице бенчмаркинга не указывается название процессора.

### ISSUE THREAD

**Issue by FenixFly** [itlab-vision/dl-benchmark#415]
При запуске скрипта inference_benchmark в Docker в результирующую таблицу не указывается название CPU, на котором происходят тесты.

Пример без докера: 
"""
"Success";"classification";"bvlcalexnet-12";"Data";"ONNX";"ONNX Runtime Python";"[1, 3, 224, 224]";"FP32";"1";"Sync";"Device: CPU, Iteration count: 2432";"CPU: Intel(R) Core(TM) i3-8100 CPU @ 3.60GHz, CPU family: x86_64, GPU: Undefined, RAM size: 32731152 kB, OS family: Linux, OS version: Linux-5.15.0-84-generic-x86_64-with-glibc2.29, Python version: 3.8.10";"0.01233";"0.01233";"81.316";"81.12";"NO_ERROR"
"""
Пример с докером:
"""
"Success";"classification";"bvlcalexnet-12";"Data";"pytorch";"ONNX Runtime Python";"[1, 3, 224, 224]";"FP32";"1";"Sync";"Device: CPU, Iteration count: 1000";"CPU family: x86_64, GPU: Undefined, RAM size: 32731152 kB, OS family: Linux, OS version: Linux-5.15.0-84-generic-x86_64-with-glibc2.29, Python version: 3.8.10";"0.01283";"0.01204";"80.023";"83.036";"NO_ERROR"
"""

Command line is: python3 /tmp/dl-benchmark/src/inference/inference_onnx_runtime.py -m /mnt/models/onnx_model_zoo/bvlcalexnet-12.onnx -i /tmp/itlab-vision-dl-benchmark-data/Datasets/ImageNET/ -b 1 -d CPU -ni 1000 --report_path /tmp/ort_python_benchmark_bvlcalexnet-12_12.10.23_01:03:40.json --raw_output true

**Comment by FenixFly** [comment]
@ismukhin тебе такое не встречалось?

**Comment by ismukhin** [comment]
> @ismukhin тебе такое не встречалось?

Я запускался только на хосте, надо будет через докер попробовать. Вы запускали это внутри контейнера, верно?

**Comment by FenixFly** [comment]
Да. Скорей всего это произошло из-за того, что из базового образа вырезали пакет openvino. Но я не уверен достаточно ли добавить в базовый образ только openvino, или openvino+openvino-dev

**Comment by valentina-kustikova** [comment]
> Да. Скорей всего это произошло из-за того, что из базового образа вырезали пакет openvino. Но я не уверен достаточно ли добавить в базовый образ только openvino, или openvino+openvino-dev

Я в докере MXNet ставила оба, чтобы получить информацию об узле.

**Comment by ismukhin** [comment]
Странно. В базовом образе ставятся сразу openvino и openvino-dev.
